external-crates/move/crates/bytecode-verifier-tests/src/unit_tests/code_unit_tests.rs

```diff
@@ -71,7 +71,7 @@ fn test_max_number_of_bytecode() {
     let module = dummy_procedure_module(nops);
 
     let result =
-        CodeUnitVerifier::verify_module(&VerifierConfig::unbounded(), &module, &mut DummyMeter);
+        CodeUnitVerifier::verify_module(&VerifierConfig::default(), &module, &mut DummyMeter);
```

Comment:
Why did a lot of these change from unbounded to default?

Reply:
Previously the config for the meter and for the verifier were coupled, and the only difference between `VerifierConfig::unbounded` and `VerifierConfig::default` were parameters that actually belong to the meter. By creating a separate `MeterConfig`, I got rid of the distinction (the parameters were unused anyway because here we're passing in a `DummyMeter`).